webpack/ForemanTasks/Components/TasksTable/TaskTableFormmatter.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
nitpick: the file name should be `TaskTableFormatters.js`

Reply:
changed, thanks